Winapp3/Beta/Winapp3.ini

```diff
@@ -327,6 +357,13 @@ Detect=HKLM\Software\JDownloader
 Default=False
 FileKey1=%ProgramFiles%\JDownloader\config|*.*
 
+[LastGood Folders *]

```

Comment:
I think a warning saying "Warning=This will delete the most recent Last Known Good Configuration information, which you will not be able to use in case of a system failure." would go good here, or something similar to it.